app/models/vm_or_template.rb

```diff
@@ -237,6 +237,8 @@ def from_infra_manager?
     'vmsafe.timeoutMS'    => [:vmsafe_timeout_ms,    :integer],
     'entitled_processors' => [:entitled_processors,  :float],
     'processor_type'      => [:processor_share_type, :string],
+    'pin_policy'          => [:pin_policy,           :string],
```

Comment:
`pin_policy` is a little vague, is this cpu pinning?  maybe `processor_pin_policy` would be more descriptive?

Reply:
pin_policy is what is coming from the `ibm-cloud-sdk-ruby`. I can change the display name to processor_pin_policy.